cetmix_tower_server/views/cx_tower_command_log_view.xml

```diff
@@ -23,12 +23,6 @@
                         bg_color="bg-danger"
                         attrs="{'invisible': ['|', ('is_running', '=', True), '&amp;', ('is_running', '=', False), ('command_status', 'in', [0, -20])]}"
                     />
-                    <widget
```

Comment:
Question: why do you want to remove this widget?

Reply:
The flight plan log didn't have that, and I removed it from the command log
I'll put it back